cmd/singularity/actions_test.go

```diff
@@ -126,6 +131,21 @@ func testSingularityRun(t *testing.T) {
 
 // testSingularityExec tests min fuctionality for singularity exec
 func testSingularityExec(t *testing.T) {
+	// Create a temp testfile
+	tmpfile, err := ioutil.TempFile("testdata", "testfile.tmp")
```

Comment:
I think it may be a bad idea to create temp files in testdata/. I think that directory should stay static with all our expected results in it.

Maybe it woud be useful to name the tempfile after the function name? So instead of `testfile.tmp` maybe `testSingularityExec.tmp`?